src/components/historyRecord/historyRecord.jsx

```diff
@@ -0,0 +1,324 @@
+                    <img src={stepStartIcon} alt="tx"></img>
+                </div>
+                <div className={`${getProgress(step, 3)}`}>
+                    <img
+                        src={
+                            txProgressIcon[
+                                `step${getProgress(
+                                    step,
+                                    2,
+                                    false
+                                )}${textTransform(from, "capitalize")}Icon`
+                            ]
+                        }
+                        alt="tx"
+                    ></img>
+                </div>
+                {hasRelay ? (
+                    <div className={`${getProgress(step, 3)}`}>
+                        <img
+                            src={
+                                txProgressIcon[
+                                    `step${getProgress(
+                                        step,
+                                        3,
+                                        false
+                                    )}RelayIcon`
+                                ]
+                            }
+                            alt="tx"
+                        ></img>
+                    </div>
+                ) : null}
+                <div className={`${getProgress(step, 4)}`}>
+                    <img
+                        src={
+                            txProgressIcon[
+                                `step${getProgress(
+                                    step,
+                                    4,
+                                    false
+                                )}${textTransform(to, "capitalize")}Icon`
+                            ]
+                        }
+                        alt="tx"
+                    ></img>
+                </div>
+            </div>
+            <div className="title-box">
+                <div>
+                    <p>{t("crosschain:Transaction Send")}</p>
+                </div>
+                <div className={`${getProgress(step, 2)}`}>
+                    <p>
+                        {t(
+                            `crosschain:${textTransform(
+                                parseChain(from),
+                                "capitalize"
+                            )} Confirmed`
+                        )}
+                    </p>
+                    <Button
+                        className="hash-btn"
+                        variant="outline-purple"
+                        target="_blank"
+                        href={getExplorerUrl(hash.from.tx, hash.from.chain)}
+                    >
+                        {t("crosschain:Txhash")}
+                    </Button>
+                </div>
+                {hasRelay ? (
+                    <div className={`${getProgress(step, 3)}`}>
+                        <p>{t(`crosschain:ChainRelay Confirmed`)}</p>
+                        {RelayButton ? RelayButton : null}
```

Comment:
dvm -> ethereum, darwinia native -> ethereum needs to use a different claim method